src/Microsoft.Health.Dicom.Core/Features/Validation/DicomElementMinimumValidation.cs

```diff
@@ -131,6 +148,42 @@ public static void ValidateUI(string value, string name)
             }
         }
 
+        public static void ValidateUI(DicomElement element)
+        {
+            EnsureArg.IsNotNull(element, nameof(element));
+            string name = GetName(element.Tag);
+            string value = element.Get<string>();
+            ValidateUI(value, name);
+        }
+
+        public static void DefaultValidate(DicomElement element)
```

Comment:
add summary

Reply:
We should not mix fo-dicom with our validation